### DISCUSSION TITLE
Headsets in ironman not working after taking over Artie Dollitle

### DISCUSSION THREAD

**Discussion by Artred42** [FakeFishGames/Barotrauma#14297] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In permadeath mode one of the players took over artie dollitle and it broke their headset (that part I'm unsure of but it's highly probable that was the case).
After that the player was not able to communicate through the headset/radio with anyone.
After I looked into the savefile character data I've noticed that his headset had TeamID="FriendlyNPC" unlike anyone else who had TeamID="Team1". After changing the value to the Team1 as everyone else had that his radio started working again.

In short: Taking over Artie Dollitle and probably all other scripted NPC's (not hired at HR but found walking around on the outpost) breaks your radio in permadeath.
 
The fix is very simple. Just change TeamID value of those NPCs headsets to "Team1" or make a trigger that changes it when they are recruited.

### Reproduction steps

1. Start a permadeath campaign
2. recruit Artie Dollitle
3. kill your friend
4. have your friend take over Artie
5. Your friend has a broken headset now.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Same will happen if you triggerevent occupationalhazards, take rudys headset
I didn't want to report it because it's too specific

Also can be fixed by just recrafting the headset
